src/features/SensorDetail/Detail/DetailSlice.ts

```diff
@@ -147,24 +148,23 @@ const DetailSelector = {
   },
 };
 
-function* fetch({ payload: { sensorId } }: PayloadAction<FetchPayload>): SagaIterator {
-  const DependencyLocator = yield getContext(DEPENDENCY.LOCATOR);
-  const sensorManager = yield call(DependencyLocator.get, DEPENDENCY.SENSOR_MANAGER);
+function* init({
```

Comment:
Changed this slice to take the initial chart data from the list state rather than requerying for it, as well as the date range